test/helpers/conditional-test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const isWindows = process.platform === 'win32';
+
+function skipSuiteOnWindows(reason) {
+  if (isWindows) {
+    test.skip(reason, () => {});
+  }
+  return isWindows;
+}
+
+module.exports.skipSuiteOnWindows = skipSuiteOnWindows;
```

Comment:
let's rename this to `isWindows` for avoid misleading

Reply:
This function does have the side effect of adding a skipped test which the name `isWindows()` would not really capture. [Jest uses the term `skipSuiteOnWindows`](https://github.com/facebook/jest/blob/master/packages/test-utils/src/ConditionalTest.ts#L30) but in this PR it's not skipping the suite, it's just skipping an individual test. What are your thoughts on `skipTestOnWindows()`?

I want to add that [I prefer Jest's way of doing this](https://github.com/facebook/jest/blob/master/packages/test-utils/src/ConditionalTest.ts#L30), but I'm not a javascript developer and I had issues getting it to work as expected. When I used `test.only(..)` the tests started failing because it wasn't shutting down the server properly. The only way I could fix it was by changing a larger portion of code than I wanted to. Perhaps you have more knowledge on how to make it so that we don't have to return a boolean, and can instead just have `skipSuiteOnWindows(..)` run `test.only(..)`.

Reply:
:+1: